client/src/constants/archiveParticipantsConstants.js

```diff
@@ -24,7 +29,6 @@ export const archiveStatusOptions = [
 ];
 
 export const archiveTypeOptions = [
-  { value: '', label: '' },
```

Comment:
Weird. Was this a workaround for an old version of the component or something? Any hints in git-blame history?

Reply:
This I'm not sure about!
I found [where it came from](https://github.com/bcgov/hcap/pull/421/files#diff-88d7257e0c015601e34d0f4e75887db2a52fbc5b045cc610efc7e2e82e2a42c6), but didn't see any comments about the empty option.

It looked super weird when I put another item at the start of the array- I nuked it after I didn't see a similar option in other select fields.

Reply:
I think this was added to have an empty option in the select list.